src/mplhep/label.py

```diff
@@ -346,3 +351,74 @@ def exp_label(
         return exptext, expsuffix, explumi
 
     return exptext, expsuffix
+
+
+def savelabels(
+    fname: str,
+    ax: plt.Axes | None = None,
+    labels: list[str] | None = None,
+    suffixes: list[str] | None = None,
+    **kwargs,
+):
+    """
+    Save multiple copies of a figure on which axes is located with the
+    most commonly used label combinations (or user supplied).
+
+    Example:
+        >>> import mplhep as hep
+        >>> hep.cms.label(data=False)
+        >>> hep.savelabels('test.png')
+
+        This will produces 4 variation on experiment label:
+            - "" -> "test_paper.png"
+            - "Preliminary" -> "test_pas.png"
+            - "Supplementary" -> "test_supp.png"
+            - "Work in Progress" -> "test_wip.png"
```

Comment:
@andrzejnovak It would also be helpful if there was an "Internal" option.

Reply:
This is eniterly manually configurable by passing `labels=[...], suffixes=[...]`. On second thought 4 might be too many for a default, maybe nothing and "Preliminary" are more sensible. 

One thing I am not sure about is whether the API would be better to pass a dict like `{'label-text':'suffix/filename', ...}` instead of two arrays.